File: twiml/voice/sim/sim-2/sim-2.5.x.cs
using Twilio.TwiML;


class Example
{
    static void Main()
    {
        var response = new VoiceResponse();
        var dial = new Dial(record: "record-from-ringing");
        dial.Sim("DE8caa2afb9d5279926619c458dc7098a8");
        response.Dial(dial);

        System.Console.WriteLine(response.ToString());
    }
}

File: twiml/voice/sim/sim-2/sim-2.5.x.php
<?php
require_once './vendor/autoload.php';
use Twilio\Twiml;

$response = new Twiml();
$dial = $response->dial(['record' => 'record-from-ringing']);
$dial->sim('DE8caa2afb9d5279926619c458dc7098a8');

echo $response;
